src/views/ApartmentApp/apartmentApp.scss

```diff
@@ -70,15 +70,16 @@
     font-size: medium;
   }
 
-  & .save-bar {
+  & .MuiGrid-item.sticky-page-bottom-bar {
```

Comment:
I like that this makes the bottom bar for starting/editing/viewing your application sticky as well. However, with that one sticky, the top one seems extremely redundant:
![image](https://user-images.githubusercontent.com/35319956/115032642-872c2680-9e97-11eb-8fc6-5337ec4547c9.png)
Is there a reason to keep the top bar around that I'm not aware of?

Reply:
I think no, there is no reason...

Reply:
I will definitely remove one of them, but I am not sure whether to have it on top or on the bottom. It will remain sticky in both cases.
I think on the bottom makes it easier to read the instructions card, and there is the nice visual aspect of it keeping the view/edit/new buttons in the same place as the save/submit, so it provides a sense of continuity for the user,

Let me know your thoughts, and any suggestions for renaming the `ApartmentHeader` if it gets moved to the bottom (or, I could possibly just migrate the features to the `BottomBar` component, which currently holds the save/submit stuff, since it is just a boolean prop that determines which one gets displayed)

Reply:
Hmm, well I'm not privy to all the requirements and use cases of the app, but I would think the bottom is nicer because, as you mentioned, it's easier to read the instructions. Keeping it at the bottom would also mean that people have to at least notice the instructions before they create a new application.

Reusing `BottomBar` would make sense to me.

Reply:
A common practice, especially since the headings on the app are our primary (gordon blue), would be to set the button to our secondary (cyan). This would visually distinguish it from the rest of the page without cluttering the site's colors.

Reply:
Nice, I'll play around with that right now